src/kakuro/core/UniquePartitions.java

```diff
@@ -3,6 +3,9 @@
 import javax.swing.tree.DefaultMutableTreeNode;
 
 /**
+ * @deprecated
+ * since iteration 2 as we are using preloaded games from the database
+ * 
```

Comment:
same here, it's just a different branch for preloading games which of course would die if it was a real kakuro puzzle because they are generated randomly otherwise it gets very boring fast as in all games

Reply:
done